rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -223,6 +229,14 @@ usb_handler_thread::usb_handler_thread()
 		usb_devices.push_back(std::make_shared<usb_device_ghltar>());
 	}
 
+	if (!found_buzz)
+	{
+		sys_usbd.notice("Adding emulated Buzz! buzzer");
+		usb_devices.push_back(std::make_shared<usb_device_buzz>(0, 3));
+		// TODO: Change to 7/8 if a buzz specific configuration window is created or a tab for the 8th player is added
```

Comment:
Most of the Buzz games have support for 8 players, even on PS2. This comment was added to explain why we have support only for 7 players.
[This is the configuration](https://user-images.githubusercontent.com/2995486/81473815-f3bf8f80-9209-11ea-9668-8ff19b288266.png) window for PCSX2, and there are 2 such windows, one for every USB port. RPCS3 needs something similar.

Reply:
This is gonna be problematic, that's why I said it's not feasible without a lot of unnecessary workarounds at this moment.
Still, I think the comment should mention the proper facts instead of the "8th player tab" stuff.
Something on the lines of:
"The current buzz emulation piggybacks on the pad input. Since there can only be 7 pads connected on a PS3 the 8th player is currently not supported."